packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -1,40 +1,38 @@
 import React, {useCallback, useMemo, useRef} from 'react';
+import PropTypes from 'prop-types';
 import Provider, {GetTranslateFromContext} from '../../atom/provider';
-import {WebContextValues} from '../../atom/provider/web-context';
 import ProgressBar from '../progress-bar';
 import {COLORS} from '../../variables/colors';
 import {createGradientBackground} from '../../util/get-background-gradient-color';
 import FaIcon from '../../atom/icon';
-import propTypes, {LearnerSkillCardProps} from './prop-types';
 import style from './style.css';
 
 const MAX_SCORE = 100;
 
-export const updateBackgroundImage = (ref: React.RefObject<HTMLDivElement>, background: string) => {
+export const updateBackgroundImage = (ref, background) => {
   if (ref.current) {
     ref.current.style.backgroundImage = background;
   }
 };
-
-const LearnerSkillCard = (props: LearnerSkillCardProps, context: WebContextValues) => {
+const LearnerSkillCard = (props, context) => {
   const {
     'aria-label': ariaLabel,
     cardIndex,
-    skillTitle,
-    skillAriaLabel,
+    title,
+    label,
     focus,
     metrics,
+    progress,
```

Comment:
```suggestion
    progress = 0,
```

peut être rajouter un default ici